scripts/convert_blipdiffusion_to_diffusers.py

```diff
@@ -0,0 +1,169 @@
+from diffusers import (
+    AutoencoderKL,
+    PNDMScheduler,
+    UNet2DConditionModel,
+)
+from transformers import CLIPTokenizer
+from src.diffusers.pipelines.blip_diffusion.modeling_blip2 import Blip2VisionModel, Blip2QFormerModel
+from src.diffusers.pipelines.blip_diffusion.modeling_ctx_clip import CtxCLIPTextModel
+from transformers.models.blip_2.configuration_blip_2 import Blip2Config, Blip2Config, Blip2VisionConfig, Blip2QFormerConfig
+from src.diffusers.pipelines import BlipDiffusionPipeline
+from LAVIS.lavis.models import load_model_and_preprocess
```

Comment:
We should either add a utility here (`is_lavis_available()`) or make it clear from the script that `LAVIS` is expected to load this model. See how it's done here: 

https://github.com/huggingface/diffusers/blob/d6141205cd69f3ad872e5a905af0b92d5f03709f/scripts/convert_original_audioldm2_to_diffusers.py#L48

Reply:
We need to build lavis from source for this, even the version in pip doesn't have blip-diffusion currently. Added comment for this in the code